kanabi.ts

```diff
@@ -25,14 +33,25 @@ type AbiStateMutability = 'view'|'external';
 type AbiFunction = {
   name: string,
   inputs: readonly AbiParameter[],
-  outputs: readonly AbiParameter[]
-}&(|{type: 'function', stateMutability?: AbiStateMutability}|
-   {type: 'constructor'});
+  output_ty: AbiType,
+  type: 'function',
+  state_mutability: AbiStateMutability,
+};
+
+
```

Comment:
nitpick: couple of spare new lines?

Reply:
Thanks Ivan, I removed the spare lines in the last commit